js/script.js

```diff
@@ -0,0 +1,222 @@
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper') 
+const form = document.getElementById('name-form') 
+
+/*audio sound*/
+let sound = document.getElementById("myAudio");
+// This function will add a chat bubble in the correct place based on who the sender is
+const showMessage = (message, sender) => {
+  if (sender === 'user') {
+    chat.innerHTML += `
+      <section class="user-msg">
+        <div class="bubble user-bubble">
+          <p>${getTime()} ${message}</p>
+        </div>
+      </section>
+    `
+  } else if (sender === 'bot') {
+    chat.innerHTML += `
+      <section class="bot-msg">
+      <div class="bubble bot-bubble">
+      <p>${getTime()} ${message}</p>
+      </div>
+      </section>
+    `
+  }
+  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  chat.scrollTop = chat.scrollHeight
+}
+// Starts here
+const greeting = () => {
+  showMessage(`Welcome to our online bootcamp, <br>What is your name?`, 'bot')
+}
+//display time
+const getTime = ()=> {
+  let today = new Date();
+  let hours =  today.getHours();
+  let minutes = today.getMinutes();
+  
+  if(hours < 10){
+    hours = "0" + hours;
+  }
+  if(minutes < 10){
+    minutes = "0" + minutes;
+  }
+  let time = hours + ":" + minutes
+   return time;
+}
+//display error message
+const errorDisplay = (errorMessage)=>{
+  inputWrapper.innerHTML =`
+  <label>${errorMessage}</label> 
+  `
+}
+
+
+//email validation
+const emailValidation = (emailMessage,courseTerm)=>{
+  let emailFormat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  if(emailMessage.match(emailFormat)){
+    setTimeout(()=>showMessage(`Thank you! You will recieve a full info-package at ${emailMessage}`, 'bot'),1000)
+    inputWrapper.innerHTML =`
+    <input type="button" value="Thanks">
+    `
+    setTimeout(()=>location.reload(), 6000)
+  }else{
+    errorDisplay(`${emailMessage} is wrong email format`)
+    setTimeout(()=>location.reload(), 6000)
+  }
+}
+
+/*take user input */
+let userText = document.getElementById("name-input");
+
+form.addEventListener("submit", (event)=> {
+  event.preventDefault();
+  userText = userText.value;
+  sound.play()
+  if (userText == ""){
+    errorDisplay ('Please enter your name')
+    setTimeout(()=> location.reload(),2000)
+  }else if(isNaN(userText) == false){
+    errorDisplay ('Please use letters only')
+    setTimeout(()=> location.reload(),2000)
+  }else{
+    showMessage(`${userText}`, 'user');
+    setTimeout(()=>  showMessage(`Thank you ${userText}<br> So I understand you want to change career?`, 'bot'),1000)
+    //Show YES NO button
+    inputWrapper.innerHTML =`
+    <input id="yes" type="button" value="YES">
+    <input id="no" type="button" value="NO">
+    `
+
+    let yesBtn = document.querySelector('#yes')
+    let noBtn = document.querySelector('#no')
+    // if choose yes
+    yesBtn.addEventListener('click', () =>{
```

Comment:
Forgot to add a space in between the arrow and the curly bracket. 

It could be written as this instead: 
`yesBtn.addEventListener('click', () => {`